src/docs/release/breaking-changes/use-maxLengthEnforcement-instead-of-maxLengthEnforced.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Use maxLengthEnforcement instead of maxLengthEnforced
+description: Introduing the MaxLengthEnforcement enum.
+---
+
+## Summary
+
+_If you don't use the `maxLength` or `maxLengthEnforced` properties in a
+`TextField`, `TextFormField`, or `CupertinoTextField`,
+or you don't care about the truncation behavior of an
+eidtable text field, you can stop reading._
+
+To control the behavior of `maxLength`
+in the `LengthLimitingTextInputFormatter`,
+use `maxLengthEnforcement` instead of `maxLengthEnforced`, which is deprecated.
+
+## Context
+
+Before [PR 63754][]: Fix TextField crashed with composing and maxLength set,
+the `LengthLimitingTextInputFormatter` always truncated the editing value,
+which resulted in engine crashes when the composing range sometimes overflowed.
+With this change, an editable `TextEditingValue` is not truncatedif only
+`maxLength` is set, but _is_ truncated after editing is complete.
+(For more specific behavior, see the [`maxLength` documentation][].)
+
+With [PR 68086][]: Introduce `MaxLengthEnforcement`,
+the truncation behavior is now controlled by passing
+`maxLengthLimitingEnforcement` to the `LengthLimitingTextInputFormatter`,
+which is then exposed to the `TextField`, `TextFormField`,
+and `CupertinoTextField` widgets.
+The `maxLengthEnforced` property is now deprecated.
+(For more information, see the docs for[`MaxLengthEnforcement`] .)
+
+The default max length truncation behavior is truncate all the time,
+which is `MaxLengthEnforcement.enforced`.
+
+## Description of change
+
+* Added a `MaxLengthLimitingEnforcement` enum, which controls all max length
+  limiting behavior.
+* The `maxLengthEnforced` property in editable text field classes is deprecated.
+* A named optional parameter, `maxLenghLimitingEnforcement` is added to
+  the `TextField`, `TextFormField`, `CupertinoTextField`,
+  and `LengthLimitingTextInputFormatter` classes, which defaults to
+  `MaxLengthLimitingEnforcement.enforced`.
+
+## Migration guide
+
+The following tips describe how to emit a warning, but not truncate,
```

Comment:
@sfshaza2 Actually warning can only show with a `TextField`, should I add `(TextField only)` after "how to emit..." ?

> The following tips describe how to emit a warning (`TextField` only), but not truncate,

Reply:
Maybe you could just say "The following tips describe how to produce common behaviors." and let the sections below elaborate.